mindmeld/components/_elasticsearch_helpers.py

```diff
@@ -107,7 +107,7 @@ def get_field_names(
             )
 
         res = es_client.indices.get(index=scoped_index_name)
-        all_field_info = res[scoped_index_name]["mappings"]["document"]["properties"]
+        all_field_info = res[scoped_index_name]["mappings"]["properties"]
```

Comment:
Is this specific for ES7 or for all ES versions?

Reply:
It's needed for a breaking change that was made in ES7. I need to check if this version will also work with previous versions of ES